acms_split.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+# Color codes
+GREEN="\033[1;32m"
+YELLOW="\033[1;33m"
+NOCOLOR="\033[0m"
+
+# Debug mode
+set -e
+set -x
+
+echo -e "${GREEN} Script Running on ${OSTYPE}${NOCOLOR}"
+
+# Sample usage: ./runsplit.sh
+# Commit will be traversed/updated from source to destination repositary.
+
+# Dynamically setting current branch to checked out branch.
+# For example if this script is run on branch, test than,
+# commit wil traverse from test branch of source repo to,
+# test branch of destination repo.
+CURRENT_BRANCH=`git rev-parse --abbrev-ref HEAD`
```

Comment:
I think we will need to change this after dev testing? Either we can pass this as parmeter to script, or provide a static value like develop. 
Currently we are dynamically setting current branch to checked out branch.
For example if this script is run on branch, test than,
commit will traverse from test branch of source repo to,
test branch of destination repo.